app/src/main/java/org/simple/clinic/patient/PatientConfig.kt

```diff
@@ -7,7 +7,11 @@ package org.simple.clinic.patient
  * */
 const val MAXIMUM_SQLITE_QUERY_LIMIT = 1000
 
-data class PatientConfig(val limitOfSearchResults: Int, val scanSimpleCardFeatureEnabled: Boolean) {
+data class PatientConfig(
+    val limitOfSearchResults: Int,
+    val scanSimpleCardFeatureEnabled: Boolean,
+    val showRecentPatients: Boolean = false
```

Comment:
Set the value of the config `showRecentPatients` in `PatientModule` from where this config is provided.